libcontainer/cgroups/fs/fs.go

```diff
@@ -207,6 +207,9 @@ func (m *manager) Apply(pid int) (err error) {
 			if cgroups.IsNotFound(err) && sys.Name() != "devices" {
```

Comment:
NIT: While it might be a bit long, I think `if cgroups.IsNotFound(err) && (c.SkipDevices || sys.Name() != "devices")` gets the intention of this logic across a little better. We also probably don't want to mask all `d.path` errors if `SkipDevices` is enabled -- just the devices cgroup ones.

Reply:
right, I'm not sure why I wrote it like this :-\